kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md

```diff
@@ -0,0 +1,227 @@
+# Sample Persistent Volume and Persistent Volume Claim
+
+The sample scripts demonstrate the creation of a Kubernetes persistent volume (PV) and persistent volume claim (PVC), which can then be used in a domain custom resource for the domain home or logs.
+
+## Prerequisites
+
+The following pre-requisites must be handled prior to running the create script:
+* Create a Kubernetes namespace for the persistent volume claim unless the intention is to use the default namespace.
+* Make sure that the host directory that will be used as the persistent volume must already exist and have the appropriate file permissions set.
```

Comment:
Make sure that the host directory that will be used as the persistent volume already exists and has the appropriate file permissions set.